onebusaway-android/src/main/res/xml/global_tracker.xml

```diff
@@ -14,7 +14,6 @@
             type="integer">120
     </item>
 
-    <string name="ga_sampleFrequency">90</string>
```

Comment:
It is still used. I deleted the redundant one. The original one is in line 20  https://github.com/OneBusAway/onebusaway-android/pull/392/files/66091e1e292b1a5e0bd2c39c82ea1e9c0d179b90#diff-e4da1a00f624408d4a24907a01d92bb6L21